### ISSUE TITLE
Runbook/Docs on services (faucet/echo)

### ISSUE THREAD

**Issue by jkrauska** [CodaProtocol/coda-automation#81]
Faucet and echo are a bit of a black box but are important pieces of testnet infra.

Need more documentation/instruction on how to manage.

**Comment by yourbuddyconner** [comment]
Opened a [PR](https://github.com/CodaProtocol/coda-automation/pull/261) today with a README for the Faucet service, @jkrauska let me know if there's anything I can add/clarify after you've had the chance to mess with it a little bit
